src/shared/sidepanel/sidepanel-support.ts

```diff
@@ -0,0 +1,3 @@
+export function isSidepanelSupported() {
+  return globalThis.chrome && 'sidePanel' in chrome;
```

Comment:
Hm, interesting, why this didn't prevent sidebar controls from appearing in Arc?

Reply:
Yeah, look like a bug in Arc